Networking Sprint Challenge - PokeDex/Networking Sprint Challenge - PokeDex/Models/PokeMon.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PokeMon.swift
+//  Networking Sprint Challenge - PokeDex
+//
+//  Created by Vijay Das on 2/1/19.
+//  Copyright © 2019 Vijay Das. All rights reserved.
+//
+
+
+// file for the PokeMon model
+
+import Foundation
+
+struct Pokemon: Codable {
+    
+    let name: String
+    let id: Int
+    let abilities: [Ability]
+    let type: [Type]
+    let sprites: SpriteImages
+
+ //   var imageData: Data?
+    
+    // need arrays for abilities, types ??
+}
+
+struct SpriteImages: Codable, Equatable {
+    let imageURL: URL
+    
+    enum CodingKeys: String, CodingKey {
+        case imageURL = "front_default"
+}
+}
+
+struct Ability: Codable,Equatable {
+    let subAbility: subAbility
```

Comment:
Make sure the name of this property matches the JSON. It should be changed to ability. The name of the struct doesn't necessarily matter, but the property name inside does.